CondTools/CTPPS/test/retrieve_sqlite_PixGainCalib_cfg.py

```diff
@@ -0,0 +1,52 @@
+import FWCore.ParameterSet.Config as cms
+import sys,os
+
+
+arguments=sys.argv
+sqlitename="ctppspixgain.db"
+tagname="CTPPSPixelGainCalibrations_test"
+runnumber=1
+outfile="output.root"
+if len(arguments)<3:
+    print "using defaults"
```

Comment:
@malbouis this is not python3 compliant, could you please fix it before we merge?